x/shield/abci.go

```diff
@@ -47,6 +49,13 @@ func EndBlocker(ctx sdk.Context, k Keeper) {
 			k.AddRewards(ctx, recipient.Provider, rewards)
 		}
 
+		// pass remaining to the pool admin if pool has ended
```

Comment:
Shouldn't this go to the last participant in the last round?

Reply:
That makes sense too, but I felt a little bit hesitant to make it biased for someone in the last position. It's a really small value (~10^(-18~-17)uctk) so it shouldn't matter much I guess

Reply:
Sure.